test/lint.ts

```diff
@@ -1,6 +1,23 @@
+/*
```

Comment:
We probably ought to use the @license annotation and the JSDoc notation here and in other test files as well. Not a blocking thing for this PR, just a thought